app/src/main/java/com/example/solanakotlincomposescaffold/usecase/MemoTransactionUseCase.kt

```diff
@@ -26,9 +25,10 @@ object MemoTransactionUseCase {
 
             // Build Message
             val blockhash = RecentBlockhashUseCase(rpcUri)
-            Message.Builder()
+            val memoTxMessage = Message.Builder()
                 .addInstruction(memoInstruction)
                 .setRecentBlockhash(blockhash)
                 .build()
```

Comment:
fyi (no change needed): there is also a `Message.toUnsignedTransaction()` method so you could do: 
```
Message.Builder()
          .addInstruction(memoInstruction)
          .setRecentBlockhash(blockhash)
          .build()
          .toUnsignedTransaction()
```